cnext_server/__main__.py

```diff
@@ -164,22 +166,43 @@ def run_help(choice):
         - cnext                        : RESUME APPLICATION or START
         - cnext 8888                   : RESUME APPLICATION at PORT 8888
         - cnext -v                     : Show the Version
+        - cnext -l on/off              : Enable/Disable Loggly Event
```

Comment:
chang the text to: `enable/disable event log`

Reply:
changed